### PR TITLE
tooltip now appears for long descriptions in summary panel

### PR DESCRIPTION
**Pull request by laclark22**
The TabularListVizulizer has now been updated so that if something is a string longer then 100 it will be turn into a subset of that string with a "..." at the end and a tooltip is created so when you put the mouse over it you can see the full description. I am planning on email Edwin to see if this is they best way to handle this or if he has another idea.  You can test this by enrolling in the patina clinical trail and looking at description in the clinical trail section.

## Submitter: Laura Clark

**Running the Application**

- [x] Manually tested in Chrome
- [x] Manually tested in IE

**Documentation**

- [x] This pull request describes why these changes were made
- [x] Recognizes any potential shortcomings/bugs in the description 
- NA Cheat sheet is updated
- NA Demo script is updated 
- NA Documentation on Wiki has been updated 
- NA Additional technical components and libraries are documented and added to wiki as needed

**NoteParser**

- NA Note parser has been updated if structured phrases change

**Code Quality**

- [x] 4-space indents - convert any tabs to spaces
- [x] Use public class field syntax for binding functions - ex. handleChange = () => { ... };
- [x] Code is commented

**Tests**

- [x] Existing tests passed
- NA Added UI tests for slim mode 
- NA Added UI tests for full mode
- NA Added backend tests for fluxNotes code
- NA Added note parser examples to test new functionality

## Reviewer 1: Name

- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

## Reviewer 2: Name

- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Comment by nicoleng12** [comment]
The code looks good. I think this is the longest tooltip we have encountered so far in our app. Because of the long description for the clinical trial, the tooltip is very tall. [NN1] Can we make the tool tip a bit wider so that it's not as tall?

File: src/summary/TabularListVisualizer.jsx [hunk 1]
```diff
@@ -297,43 +299,64 @@ class TabularListVisualizer extends Component {
...
+                    }
+
+                    if(!Lang.isNull(elementText) && elementText.length > 100){
+                        const text = <span>{longElementText}</span>
+                        columnItem = (
+                            <Tooltip
+                                key={elementId}
+                                overlayStyle={{'visibility': true}}
+                                placement="top"
+                                overlayClassName={`context-panel-tooltip large`}
```
Comment (danlee1025) [inline src/summary/TabularListVisualizer.jsx]: [DL1] Should we use a different overlayClassName here since this Tooltip is in the targeted data panel?

**Comment by gregquinn2001** [comment]
Looks good to me but NN1 and DL1 sound good. I don't think we need to spend a lot more time on this though so only if they're easy!

**Comment by laclark22** [comment]
Addressed DL1 and NN1.

**Comment by laclark22** [comment]
rebased on master

**Comment by nicoleng12** [comment]
This looks great! I'm just running tests now and once those pass I'll approve and merge this
